code/modules/projectiles/projectile/special/mindflayer.dm

```diff
@@ -1,9 +1,18 @@
 /obj/item/projectile/beam/mindflayer
 	name = "flayer ray"
+	icon_state = "flayerlaser"
+	damage = 8
```

Comment:
this is going to heal way too fast

Reply:
You say this like it would be a bad thing